packages/playground/src/components/Playground.tsx

```diff
@@ -11,129 +11,18 @@ import {
-    focusOnFirstError: { type: 'boolean', title: 'Focus on 1st Error' },
-    showErrorList: {
-      type: 'string',
-      default: 'top',
-      title: 'Show Error List',
-      enum: [false, 'top', 'bottom'],
-    },
-  },
-};
-
-type EditorsProps = {
-  schema: RJSFSchema;
-  setSchema: React.Dispatch<React.SetStateAction<RJSFSchema>>;
-  uiSchema: UiSchema;
-  setUiSchema: React.Dispatch<React.SetStateAction<UiSchema>>;
-  formData: any;
-  setFormData: React.Dispatch<React.SetStateAction<any>>;
-  extraErrors: ErrorSchema | undefined;
-  setExtraErrors: React.Dispatch<React.SetStateAction<ErrorSchema | undefined>>;
-  setShareURL: React.Dispatch<React.SetStateAction<string | null>>;
-};
-
-function Editors({
-  extraErrors,
-  formData,
-  schema,
-  uiSchema,
-  setExtraErrors,
-  setFormData,
-  setSchema,
-  setShareURL,
-  setUiSchema,
-}: EditorsProps) {
-  const onSchemaEdited = useCallback(
-    (newSchema) => {
-      setSchema(newSchema);
-      setShareURL(null);
-    },
-    [setSchema, setShareURL]
-  );
-
-  const onUISchemaEdited = useCallback(
-    (newUiSchema) => {
-      setUiSchema(newUiSchema);
-      setShareURL(null);
-    },
-    [setUiSchema, setShareURL]
-  );
-
-  const onFormDataEdited = useCallback(
-    (newFormData) => {
-      if (
-        !isEqualWith(newFormData, formData, (newValue, oldValue) => {
-          // Since this is coming from the editor which uses JSON.stringify to trim undefined values compare the values
-          // using JSON.stringify to see if the trimmed formData is the same as the untrimmed state
-          // Sometimes passing the trimmed value back into the Form causes the defaults to be improperly assigned
-          return JSON.stringify(oldValue) === JSON.stringify(newValue);
-        })
-      ) {
-        setFormData(newFormData);
-        setShareURL(null);
-      }
-    },
-    [formData, setFormData, setShareURL]
-  );
-
-  const onExtraErrorsEdited = useCallback(
-    (newExtraErrors) => {
-      setExtraErrors(newExtraErrors);
-      setShareURL(null);
-    },
-    [setExtraErrors, setShareURL]
-  );
 
-  return (
-    <div className='col-sm-7'>
-      <Editor title='JSONSchema' code={toJson(schema)} onChange={onSchemaEdited} />
-      <div className='row'>
-        <div className='col-sm-6'>
-          <Editor title='UISchema' code={toJson(uiSchema)} onChange={onUISchemaEdited} />
-        </div>
-        <div className='col-sm-6'>
-          <Editor title='formData' code={toJson(formData)} onChange={onFormDataEdited} />
-        </div>
-      </div>
-      {extraErrors && (
-        <div className='row'>
-          <div className='col'>
-            <Editor title='extraErrors' code={toJson(extraErrors || {})} onChange={onExtraErrorsEdited} />
-          </div>
-        </div>
-      )}
-    </div>
-  );
-}
-
-interface LiveSettings {
+export interface LiveSettings {
   showErrorList: false | 'top' | 'bottom';
   [key: string]: any;
 }
```

Comment:
I would strongly suggest moving this into the `Header` rather than leaving it here because you are creating a circular import reference between the two files that might cause future issues.

```suggestion
```